casper/src/main/scala/coop/rchain/casper/ReportingCasper.scala

```diff
@@ -220,338 +163,97 @@ object ReportingCasper {
                    }
         } yield result
 
+      private def replayDeploys(
+          runtime: ReportingRuntime[F],
+          startHash: StateHash,
+          terms: Seq[ProcessedDeploy],
+          isGenesis: Boolean
+      ): F[Either[ReplayFailure, List[(ProcessedDeploy, Seq[Seq[ReportingEvent]])]]] =
+        (for {
+          _ <- EitherT.right(runtime.reset(Blake2b256Hash.fromByteString(startHash)))
+          res <- EitherT.right(terms.toList.traverse { term =>
+                  for {
+                    rd <- runtime
+                           .replayDeployE(isGenesis)(term)
```

Comment:
Does behavior of replay changed to not replay cost accounting? Or genesis parameter is used differently?

Reply:
Good catch~!